R/intervals.R

```diff
@@ -87,3 +87,28 @@ get_promoters <- function(upstream = 500, downstream = 50) {
         mutate(start = ifelse(strand == 1, start - upstream, start - downstream), end = ifelse(strand == 1, end + downstream, end + upstream)) %>%
         gintervals.force_range()
 }
+
+#' Convert 10X-format peak names to misha intervals
+#' @param pn peak names as generated by 10X ATAC pipeline ("{chr}-{start}-{end}" format)
```

Comment:
Isn't the format chr:start-end? It was like this in the multiome you pointed me to.

Reply:
It is, added this to the regex